src/components/pages/home/elements/WeatherBlock.js

```diff
@@ -1,12 +1,44 @@
 import React, { Component } from 'react';
+import WeatherTitle from './weatherTitle/WeatherTitle';
+import WeathewrAttributes from './weatherAttributes/WeatherAttributes';
+import getMarsWeatherData from '../../../../api/inSightApiClient';
 import '../HomePage.css';
 import PageNavButton from '../../../shared/PageNavButton';
 
 export default class WeatherBlock extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { weather: this.dummyValues() };
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  getData() {
+    getMarsWeatherData().then(weather => (this.setState({
+      weather: weather.slice(weather.length - 1)[0],
+    })
+    ));
+  }
+
+  dummyValues() {
+    return ({
+      earthDate: null,
+      sol: null,
+      pressure: {avg: null, max: null, min: null},
+      season: null,
+      temp: {avg: null, max: null, min: null},
+      wind: {avg: null, max: null, min: null, dir: null},
+    });
+  }
+
   render() {
+    console.log(this.state.weather);
     return (
       <div className="weather-block">
-        <h3>Today&apos;s weather forecast: </h3>
+        <WeatherTitle MarsSol={this.state.weather.sol} EarthDate={this.state.weather.earthDate} MarsSeason={this.state.weather.season} />
+        <WeathewrAttributes Temp={this.state.weather.temp} Pressure={this.state.weather.pressure} Wind={this.state.weather.wind} />
```

Comment:
These property variables should all be in `camelCase` 🐪

Also, when we've got a lot of properties to pass down the elements should be formatted as multiple lines, e.g.

```
 <WeatherTitle 
     marsSol={this.state.weather.sol}
     earthDate={this.state.weather.earthDate}
     marsSeason={this.state.weather.season}
/>
```